plat/common/include/pci/pci_bus.h

```diff
@@ -247,6 +251,30 @@ static struct pci_bus_handler ph __unused;
 #define PCI_CONF_IOBAR_SHFT         (0x0)
 #define PCI_CONF_IOBAR_MASK         (~0x3)
 
+#define PCI_CONF_BAR0              (0x10)
+#define PCI_CONF_BAR0_SHFT         (0)
+#define PCI_CONF_BAR0_MASK         (0xFFFFFFFF)
```

Comment:
Ditto. Please be more specific with commits and commits messages, this is too general.
This goes in its own commit as well e.g. `plat/common/pci: Add macro's for PCI BARs`.